tests/PythonTest.sh

```diff
@@ -53,9 +53,9 @@ function run_tests() {
 }
 
 # Run test suite with these interpreters. The arguments are benchmark counts.
-run_tests python2.6 100 100 100 100 false
-run_tests python2.7 100 100 100 100 false
-run_tests python2.7 100 100 100 100 true
+# run_tests python2.6 100 100 100 100 false
```

Comment:
i guess if it is decided to remove Python2 support this can be deleted and not just commented out